resources/api/examples/tieturvallisuustarkastuksen-kirjaus-request.json

```diff
@@ -0,0 +1,43 @@
+{
+  "otsikko": {
+    "lahettaja": {
+      "jarjestelma": "Urakoitsijan järjestelmä",
+      "organisaatio": {
+        "nimi": "Urakoitsija",
+        "ytunnus": "1234567-8"
+      }
+    },
+    "viestintunniste": {
+      "id": 124
+    },
+    "lahetysaika": "2023-07-30T12:00:00+02:00"
```

Comment:
Miltä se tuntuisi, jos rajapinnoissa siirryttäisiin vaivihkaa kokonaan UTC:n käyttöön? Eli ymmärtäisin sen niin, että vaadittaisiin timestampit tyyliin näin: "2023-07-30T09:00:00Z"